proteinbenchmark/protein_system.py

```diff
@@ -383,12 +498,21 @@ def analyze_observables(self, replica: int = 1):
 
             replica_dir = Path(self.base_path, f"replica-{replica:d}")
             replica_prefix = Path(replica_dir, self.system_name)
-
+            
+            if self.sim_platform != 'gmx':
+                traj_path = f"{replica_prefix}-production.dcd"
+                output_selection = 'chainid == "A"' 
+                topology_path = self.minimized_pdb
+            else:
+                traj_path = f"{replica_dir}/traj.xtc"
+                output_selection = 'resname != "HOH" && resname != "NA"'
```

Comment:
Also exclude chloride ions. This doesn't matter for GB3, but it will for other benchmark targets.